app/cluster/integration_test.go

```diff
@@ -249,7 +248,10 @@ func TestDynamicClusterManager(t *testing.T) {
 
 	wait := make(chan bool)
 	go func() {
-		dCM.Run(ctx)
+		err := dCM.Run(ctx)
+		if err != nil {
+			log.Printf("cluster runner stopped: %s \n", err)
```

Comment:
Can we use `t.Logf` inside tests please? They respect `testing.Verbose()` and don't require the trailing `\n`.

Reply:
Nice idea! Thx dude!